src/storage/src/hummock/state_store.rs

```diff
@@ -544,11 +597,11 @@ impl StateStore for HummockStorage {
         async move { Ok(self.local_version_manager.wait_epoch(epoch).await?) }
     }
 
-    fn sync(&self, epoch: Option<u64>) -> Self::SyncFuture<'_> {
+    fn sync(&self, epoch_range: (HummockEpoch, HummockEpoch)) -> Self::SyncFuture<'_> {
```

Comment:
Is range must starts from the next epoch of last synced epoch? If so, can `StateStore` record it inner to avoid pass two given epoch?

Reply:
Besides, IIRC, `epoch` is not continuous, it is combined by timestamp and sequence.